src/assets/wise5/authoringTool/node/node-authoring/node-authoring.component.ts

```diff
@@ -555,28 +523,14 @@ class NodeAuthoringController {
     return componentObjects;
   }
 
-  showComponentAdvancedAuthoring(componentContent: ComponentContent, event: any) {
+  showComponentAdvancedAuthoring(componentContent: ComponentContent, event: any): void {
     event.stopPropagation();
-    const component = new Component(componentContent, this.nodeId);
-    this.$mdDialog.show({
-      templateUrl: 'assets/wise5/authoringTool/components/edit-component-advanced.html',
-      controller: [
-        '$scope',
-        '$mdDialog',
-        function ($scope: any, $mdDialog: any) {
-          $scope.close = function () {
-            $mdDialog.hide();
-          };
-        }
-      ],
-      controllerAs: '$ctrl',
-      bindToController: true,
-      locals: {
-        component: component,
+    this.dialog.open(EditComponentAdvancedComponent, {
+      data: {
+        component: new WiseComponent(componentContent, this.nodeId),
         nodeId: this.nodeId
       },
```

Comment:
No need to pass in nodeId again since we can get that from the component?
```suggestion
      data: new WiseComponent(componentContent, this.nodeId),
```